model/Hierarchy.php

```diff
@@ -617,9 +617,14 @@ public function numChildren($cache = true) {
 	 */
 	public function stageChildren($showAll = false) {
 		$baseClass = ClassInfo::baseDataClass($this->owner->class);
+		$hidden = array();
+		if ($baseClass::config()->get('hideFromHierarchy')) {
+			$hidden = $baseClass::config()->get('hideFromHierarchy');
```

Comment:
Could you please document this feature? I suggest the following in Hierachy.php itself

```php

/**
 * A list of classnames to exclude from display in the hierarchy.
 * Note: Does not filter subclasses (non-inheriting)
 *
 * @var array
 * @config
 */
private static $hide_from_hierarchy = array();
```

I suggest to name the variable hide_from_hierarchy as it meets the static variable naming convention.